qmi/instruments/tenma/psu_72.py

```diff
@@ -0,0 +1,490 @@
+
+        Parameters:
+            voltage: Voltage setting for the channel in Volts.
+            output_channel: The channel to send the inquiry to.
+
+        Raises:
+            ValueError: If the input voltage setting is not between 0 and self.MAX_VOLTAGE
+        """
+        if not 0 <= voltage <= self.MAX_VOLTAGE:
+            raise ValueError(f"Invalid current setting {voltage}, not between 0 and {self.MAX_VOLTAGE} V")
+
+        ch = output_channel or ""
+        command = f"VSET{ch}:{voltage:.3f}"
+        self._send(command)
+
+    @rpc_method
+    def enable_output(self, output: bool) -> None:
+        """Enable or disable output from the PSU.
+
+        Parameters:
+            output: Boolean value to either set output ON (True) or OFF (False).
+        """
+        raise NotImplementedError()
+
+
+class Tenma72_2550(Tenma72_Base):
+    """Instrument driver for the Tenma 72-2550. The driver is tested with this model, but the respective
+    manual is also for models 72-2535, 72-2540, 72-2545, 72-2925, 72-2930, 72-2935, 72-2940 & 72-10480.
+
+    This driver can be used only with (USB-to-)serial communications.
+
+    Attributes:
+        DEFAULT_BAUDRATE: The default baudrate in case of serial connection is used (Baud).
+        MAX_VOLTAGE:      The maximum voltage that can be set with the PSU (Volts).
+        MAX_CURRENT:      The maximum current that can be set with the PSU (Amperes).
+    """
+    DEFAULT_BAUDRATE = 9600
+    MAX_VOLTAGE = 60
+    MAX_CURRENT = 3
+
+    @rpc_method
+    def get_status(self) -> Dict[str, Any]:
+        """Get the power supply status as a dictionary of status values.
+
+        Dictionary composition is:
+            Ch1Mode: "C.V" | "C.C"
+            Ch2Mode: "C.V" | "C.C"
+            Tracking: xx, where xx is byte
+                * 00 = Independent
+                * 01 = Tracking series
+                * 10 = Tracking parallel
+            OutputEnabled: True | False
+        
+        Returns: 
+            Dictionary of status values.
+        """
+        self._send("STATUS?")
+        statusByte = ord(self._transport.read(1, 1))  # Read response byte
+        ch1mode = (statusByte & 0x01)
+        ch2mode = (statusByte & 0x02)
+        tracking = (statusByte & 0x0C) >> 2
+        output_enabled = (statusByte & 0x40)
+
+        if tracking == 0:
+            tracking = "Independent"
+        elif tracking == 1:
+            tracking = "Tracking Series"
+        elif tracking == 2:
+            tracking = "Tracking Parallel"
+        else:
+            tracking = "Unknown"
+        return {
+            "Ch1Mode": "C.V" if ch1mode else "C.C",
+            "Ch2Mode": "C.V" if ch2mode else "C.C",
+            "Tracking": tracking,
+            "OutputEnabled": bool(output_enabled),
+        }
+
+    @rpc_method
+    def enable_output(self, output: bool) -> None:
+        command = f"OUT{int(output)}"
+        self._enable_output(output, command)
+
+
+class Tenma72_2535(Tenma72_2550):
```

Comment:
class dostrings here and for the rest of the inheriting classes would be nice (even if its one line). Maybe you can mention in the docstring what the values of the attributes are, so then our sphinx documentation can also catch it.

Reply:
Ah, good point! I'll edit those.